packages/blockchain-wallet-v4-frontend/src/assets/locales/en.json

```diff
@@ -1112,6 +1112,8 @@
   "modals.simplebuy.save_my_card": "Save My Card",
   "modals.simplebuy.selectcurrency": "Select Your Currency",
   "modals.simplebuy.cryptoselect": "Buy with Cash or Card",
+  "modals.simplebuy.paymentmethods": "Payment Methods",
+  "modals.simplebuy.depositcash_description": "Send funds directly from your bank to your Blockchain.com Wallet. Once we receive the manual transfer, use that cash to buy crypto.",
```

Comment:
This file is generated automatically, no need to modify it directly.

Reply:
oh didn't been aware, cool!